src/backend/aspen/database/models/usher.py

```diff
@@ -14,6 +16,8 @@ class UsherOption(idbase, DictMixin):  # type: ignore
     value = Column(String, unique=True, nullable=False)
     # `priority` is order we display options to user. LOWEST number is max priority.
     priority = Column(Integer, unique=True)
```

Comment:
I saw you dropped the specific, `priority` only unique constraint in the migrations -- which is great, I was definitely thinking that would be a good thing to do as I read through the PR -- can you also drop the `unique=True` here in the model for that column?

The migration also adds a multi-col unique constraint, so can you also add a `__table_args__` to match that here in the model as well please? Something like this:
```
    __table_args__ = (
        UniqueConstraint(
            "sample_id",
            "mutations_caller",
        ),
    )
```

I definitely think you did the right thing in your migration, just want to keep the model in sync with the DB.

Reply:
Right, I overlooked that. Thanks for catching it!